### PR TITLE
Test publish

### PR DESCRIPTION
**Pull request by chainlink**
This adds publish to testpypi on tag push

To test tag push, we need to push this code to master

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -34,6 +34,30 @@ jobs:
...
+  test-publish:
+    docker:
+      - image: circleci/python:3.7
+    steps:
+      - checkout
+      - run:
+          name: Init .pypirc
+          command: |
+            echo -e "[testpypi]" >> ~/.pypirc
+            echo -e "username = $PYPI_USERNAME" >> ~/.pypirc
```
Comment (albscui) [inline .circleci/config.yml:46]:
```suggestion
            echo -e "username = $TEST_PYPI_USERNAME" >> ~/.pypirc
```

File: .circleci/config.yml [hunk 2]
```diff
@@ -34,6 +34,30 @@ jobs:
...
+            echo -e "password = $PYPI_PASSWORD" >> ~/.pypirc
+      - run:
+          name: Install deps
+          command: |
+            pip install twine
```
Comment (albscui) [inline .circleci/config.yml:51]:
```suggestion
            python3 -m pip install twine
```

**Review by albscui — APPROVED** [review]
Some small suggestions, but LGTM!

File: .circleci/config.yml (hunk 1 above)
Comment (albscui) [reply .circleci/config.yml:46]: Although the username would most likely always be `__token__`. I'm assuming eventually we would have two sets of env variables in Circle. `TEST_PYPI_PASSWORD` and `PYPI_PASSWORD`

Comment (chainlink) [reply .circleci/config.yml:46]: yes, these are set

Comment (albscui) [reply .circleci/config.yml:46]: I see, my assumption was that env variables are set globally for a project. So if we have two jobs one for testpypi and another for real pypi, then wouldn't we need both `TEST_PYPI_PASSWORD` and `PYPI_PASSWORD`?

**Pull request merged**